renpy/common/00layeredimage.rpy

```diff
@@ -145,8 +145,7 @@ python early in layeredimage:
 
             d = renpy.displayable(d)
```

Comment:
This can also be emitted as it's the first thing that `At` will do.

Reply:
Only if at least one of self.at, self.group_args and self.transform_args is non-empty. In other cases this changes the return type of the method. I'm not sure of the consequences of that, so I'd abstain.

Reply:
I don't follow. The call to `displayable` inside `At` isn't contingent on anything. You could call `At(d)` and it would still call `displayable`. :/